src/js/components/SelectMultiple/SelectMultiple.js

```diff
@@ -126,6 +128,19 @@ const SelectMultiple = forwardRef(
       initialValue: defaultValue || '',
     });
 
+    const [ariaLabelledBy, setAriaLabelledBy] = useState();
```

Comment:
Any chance we could do this as a useMemo instead of a setState + useEffect to avoid extra re-render from the useEffect?

Reply:
good call!

Reply:
from co-pilot

I recommend Option 1 (going back to useEffect) because:

The DOM query needs to happen after the label element is rendered
FormField renders the label, then the Select component
useEffect ensures the DOM is ready
The extra re-render is minimal compared to the complexity of timing the DOM query correctly

Reply:
It feels like we'd want to rethink a variety of things in FormField logic in the future, but this at least feels like an improvement on what we have currently.

Reply:
We can see if Mike has any remaining concerns or if he's okay with the approach.

Reply:
okay I added the check to see if the child `id` matches the `htmlFor`